addons/ballistics/CfgMagazines.hpp

```diff
@@ -237,17 +240,35 @@ class CfgMagazines {
     class ACE_30Rnd_65x47_Scenar_mag: 30Rnd_65x39_caseless_mag {
         author = ECSTRING(common,ACETeam);
         ammo = "ACE_65x47_Ball_Scenar";
+        initSpeed = 761;
         displayName = CSTRING(30Rnd_65x47_Scenar_mag_Name);
         displayNameShort = CSTRING(30Rnd_65x47_Scenar_mag_NameShort);
         descriptionShort = CSTRING(30Rnd_65x47_Scenar_mag_Description);
     };
+    class ACE_20Rnd_65x47_Scenar_mag: 20Rnd_650x39_Cased_Mag_F {
+        author = ECSTRING(common,ACETeam);
+        ammo = "ACE_65x47_Ball_Scenar";
+        initSpeed = 779;
+        displayName = CSTRING(20Rnd_65x47_Scenar_mag_Name);
+        displayNameShort = CSTRING(20Rnd_65x47_Scenar_mag_NameShort);
+        descriptionShort = CSTRING(20Rnd_65x47_Scenar_mag_Description);
+    };
     class ACE_30Rnd_65_Creedmor_mag: 30Rnd_65x39_caseless_mag {
         author = ECSTRING(common,ACETeam);
         ammo = "ACE_65_Creedmor_Ball";
+        initSpeed = 815;
         displayName = CSTRING(30Rnd_65_Creedmor_mag_Name);
         displayNameShort = CSTRING(30Rnd_65_Creedmor_mag_NameShort);
         descriptionShort = CSTRING(30Rnd_65_Creedmor_mag_Description);
     };
+    class ACE_20Rnd_65_Creedmor_mag: 20Rnd_650x39_Cased_Mag_F {
+        author = ECSTRING(common,ACETeam);
+        ammo = "ACE_65_Creedmor_Ball";
+        initSpeed = 808;
```

Comment:
Also, just looking at the data from that test that was done it seems like the base speed (if we're considering a 16" barrel the base speed for the 6.5mm cartriges) should be around 760 for the creedmoor and lapua ammo.